_docs/tasks/installing-istio.md

```diff
@@ -29,6 +29,13 @@ This page shows how to install and configure Istio in a Kubernetes cluster.
   ```bash
   $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
   ```
+* If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
+
```

Comment:
... uid 0. You ...

... enable this for Istio's ingress and egress service accounts as ...